src/middlewares/logger.ts

```diff
@@ -0,0 +1,30 @@
+import {createLogger, format, transports} from 'winston';
+import {Context} from 'koa';
+
+const {combine, timestamp, printf} = format;
+
+// tslint:disable-next-line:no-shadowed-variable
+const myFormat = printf(({level, message, timestamp}) => {
+    return `${timestamp} ${level}: ${message}`;
+});
+
+export const logger = createLogger({
```

Comment:
logger its not a middleware

Reply:
seperated the logger to another folder and added logging-middleware